exe/Main.hs

```diff
@@ -44,14 +50,38 @@ import qualified System.Directory.Extra as IO
 import System.Environment
 import System.IO
 import System.Exit
+import HIE.Bios.Environment (addCmdOpts)
 import Paths_ghcide
 import Development.GitRev
-import Development.Shake (Action, Rules, action)
+import Development.Shake (Action,  action)
 import qualified Data.HashSet as HashSet
+import qualified Data.HashMap.Strict as HM
 import qualified Data.Map.Strict as Map
-import HIE.Bios
-import Rules
-import RuleTypes
+import Data.Either
+--import Outputable (pprTraceM, ppr, text)
+import qualified Crypto.Hash.SHA1               as H
+import qualified Data.ByteString.Char8          as B
+import           Data.ByteString.Base16         (encode)
```

Comment:
Please don’t align imports. It just makes diffs more noisy.

Reply:
Ok, but there are other places where imports are already aligned..

Reply:
Yep, the formatting on this code reveals the history of multiple different authors. One day it would be great to run a real formatter over it, since that's the only way to recover from the mess we have now.